# Let the plugin makefile system work with our install system: patch the
# directory path to correspond with our install directory
# 01/02/2016 - wpoely86@gmail.com
diff -ur psi4-9837d4211dfee0006a9810b97f507af2883fa9fc.orig/CMakeLists.txt psi4-9837d4211dfee0006a9810b97f507af2883fa9fc/CMakeLists.txt 
--- psi4-9837d4211dfee0006a9810b97f507af2883fa9fc.orig/CMakeLists.txt	2016-06-30 04:10:18.467252617 +0200
+++ psi4-9837d4211dfee0006a9810b97f507af2883fa9fc/CMakeLists.txt	2016-06-30 04:12:21.609231000 +0200
@@ -296,6 +296,7 @@
 foreach(dir ${dirs})
     set(PLUGIN_INCLUDE_DIRECTORIES "-I${dir} ${PLUGIN_INCLUDE_DIRECTORIES}")
 endforeach()
+set(PLUGIN_INCLUDE_DIRECTORIES "-I$ENV{PSI_SRC_INSTALL_DIR}/include -I$ENV{PSI_SRC_INSTALL_DIR}/src/lib/ -I$ENV{PSI_OBJ_INSTALL_DIR}/include -I$ENV{PSI_OBJ_INSTALL_DIR}/src/lib/ ${PLUGIN_INCLUDE_DIRECTORIES}")
 foreach(def ${defs})
     set(PLUGIN_DEFINES "-D${def} ${PLUGIN_DEFINES}")
 endforeach()
